assets/app/scripts/directives/podStatusChart.js

```diff
@@ -109,18 +119,11 @@ angular.module('openshiftConsole')
             }
           });
 
-          angular.forEach(phases, function(phase) {
-            columns.push([phase, countByPhase[phase] || 0]);
-          });
-
-          chart.load({ columns: columns });
+          return countByPhase;
         }
 
-        $timeout(function() {
-          chart = c3.generate(config);
-          $scope.$watch('pods', updateChart, true);
-          $scope.$watch('desired', updateCenterText);
-        }, 0, false);
+        $scope.$watch(countPodPhases, updateChart, true);
```

Comment:
how often is this causing countPodPhases to get run, wouldn't it be better to update a podPhases variable on the scope when we get watch events for pods?